docs/maps/search.asciidoc

```diff
@@ -43,6 +80,18 @@ This can also occur with a single layer with an {es} source and a <<terms-join>>
 
 Searching across multiple indices might sometimes result in empty layers.
 The most common cause for empty layers are searches for a field that exists in one index, but does not exist in other indices.
+
+[float]
+[[maps-disable-search-for-layer]]
+==== Disable search for layer
+
+You can disable the globar search bar from applying search context to a layer by unchecking *Apply global filter to layer* in Layer settings.
```

Comment:
```suggestion
To prevent the global search bar from applying search context to a layer, clear the *Apply global filter to layer* checkbox in Layer settings.
```